src/kmer_unordered_set.cpp

```diff
@@ -47,58 +68,49 @@ void KmerSets::get_parent_kmer_sets(){
     path relative_hap2_kmer_list_path = "data/hg04.all.homo.unique.kmer.1000.fa";
     path absolute_hap2_kmer_list_path = project_directory / relative_hap2_kmer_list_path;
 
-    load_file_into_unordered_set(absolute_hap1_kmer_list_path,dad_kmer_set);
-    load_file_into_unordered_set(absolute_hap2_kmer_list_path,mom_kmer_set);
-
-}
-
-void KmerSets::fill_kmer_sets() {
-	// hard code filled 
-	mom_kmer_set.insert("GATTATTTTTACATTAAGGTTATCACCTCAAATCCTTTTTTAAAAATAGTCAGCA");
-	mom_kmer_set.insert("GATTATTTTTACATTAAGGTTATCACCTCAAATCCTTTTTTAAAAATAGTCAGCC");
-	mom_kmer_set.insert("GATTATTTTTACATTAAGGTTATCACCTCAAATCCTTTTTTAAAAATAGTCAGCT"); // child match
-	
-	dad_kmer_set.insert("AACTCTTCCCAGATGGATTTGAAACTTGAAATATGGAGGATAGAACTGTTACTGC");
-	dad_kmer_set.insert("CACTCTTCCCAGATGGATTTGAAACTTGAAATATGGAGGATAGAACTGTTACTGC");
-	dad_kmer_set.insert("GACTCTTCCCAGATGGATTTGAAACTTGAAATATGGAGGATAGAACTGTTACTGC"); // child match
+    load_file_into_unordered_set(absolute_hap1_kmer_list_path,hap1_kmer_set);
+    load_file_into_unordered_set(absolute_hap2_kmer_list_path,hap2_kmer_set);
 
 }
 
-// make a single kmer find
+// single kmer find
 bool KmerSets::find_haplotype_single_kmer_count(string child_kmer) {
-	mom_kmer_count = 0;
-	dad_kmer_count = 0;
+	hap1_kmer_count = 0;
+	hap2_kmer_count = 0;
 
-	if (mom_kmer_set.find(child_kmer) != mom_kmer_set.end()){
-		mom_kmer_count++;
+	if (hap1_kmer_set.find(child_kmer) != hap1_kmer_set.end()){
+		hap1_kmer_count++;
 		}
 
-	if (dad_kmer_set.find(child_kmer) != dad_kmer_set.end()){
-		dad_kmer_count++;		}
+	if (hap2_kmer_set.find(child_kmer) != hap2_kmer_set.end()){
+		hap2_kmer_count++;		
+		}
 
 	cout << " Checking single kmer: " << child_kmer << endl;
-	cout << " found " << mom_kmer_count << " mom kmers." << endl ;
-	cout << " found " << dad_kmer_count << " dad kmers." << endl << endl ;
+	cout << " found " << hap1_kmer_count << " hap1 (hg03) kmers." << endl ;
+	cout << " found " << hap2_kmer_count << " hap2 (hg04) kmers." << endl << endl ;
 	return 0;
```

Comment:
lets figure out a way to do this outside of this function. Ideally we just return a `pair<bool,bool>` that contains `{hap0_found,hap1_found}`

Reply:
and then you can do the printing in the main executable (generally safer than printing inside a class method that will be used many times)

Reply:
I am working on this now